r/R/trackplots.R

```diff
@@ -450,6 +478,126 @@ trackplot_gene <- function(transcripts, region, exon_size = 2.5, gene_size = 0.5
   wrap_trackplot(plot, height=ggplot2::unit(1, "null"))
 }
 
+#' Plot peak tracks
+#' @param loci `r document_granges("Genomic loci")`
+#' @inheritParams trackplot_coverage
+#' @param annotation_size size for peak annotations
+#' @param label_by Name of a metadata column in `loci` to use for labeling, or a numeric vector with same length as loci  Column must be type character, factor or numeric
+#' @param label_size size for transcript labels in units of mm
+#' @param color_by Name of a metadata column in `loci` to use for coloring, or a numeric vector with same length as loci.  Column must be type character, factor, or numeric
+#' @param colors Vector of hex color codes to use for the color gradient
+#' @param direction_by Name of a metadata column in `loci` to use for determining direction of arrows
```

Comment:
I'd agree, I addressed it with more flexibility because I wasn't sure if there was additional biological context that I was missing

Reply:
It's definitely a good instinct to keep the argument flexible since we'll need one argument either way. In this case I think it'll be pretty standard to encode direction as strand (it seems to be what Selin did in their binding-site bed files)